### ISSUE TITLE
When a node is added with connecting edge, its edge should inherit all properties of the edge for the parent node. 

### ISSUE THREAD

**Issue by raindropsfromsky** [jgraph/drawio#2818]
* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Is your feature request related to a problem? Please describe.**
Let's say I am drawing a mindmap. If I add a subnode or a sibling node with a connecting edge, this edge has different properties from the edges I started out with. (I have no clue why the properties change.) I have to separately select all edges and change their properties. Copying style from the previous edge and pasting it on new edges does NOT work!

**Describe the solution you'd like**
The diagram should remember the whole set of properties for the last edge created, and go on creating new edges with the same properties. 

If I select specific _existing_ edges and change their properties, these properties need not be inherited when new nodes are connected to that node.

**Describe alternatives you've considered**
Currently, the definition of "style" for an edge does not include its properties.
Thus even if I copy the style from one edge and paste it on another, the other edge may still look completely different, because its properties are different.

Therefore, there should be a way to copy and paste _all_ properties from one edge to another.

I have already raised a separate bug for this (#1956), but it was closed without providing any reason! :(

**Additional context**

**Comment by davidjgraph** [comment]
Reopened other issue, closing this as duplicate of #1956

**Comment by raindropsfromsky** [comment]
IMHo this bug should not be counted as duplicate of #1956, because both are asking for different functionalities:

This bug is about continuing the edge style in all subsequently created edges. <br>In other words, every edge should inherit _all the properties_ of the predecessor edge _by default,_ without a copy-paste operation.

In contrast, bug #1956 is about **copy-paste of styles** from any edge to any edge. 
In other words, both are existing edges, and we should be able to copy all properties from one edge to another.

Therefore, this bug should be re-opened.
